documentation/sphinx/source/api-python.rst

```diff
@@ -26,6 +26,7 @@
 .. |max-retry-delay-database-option| replace:: :func:`Database.options.set_transaction_max_retry_delay`
 .. |transaction-size-limit-database-option| replace:: :func:`Database.options.set_transaction_size_limit`
 .. |causal-read-risky-database-option| replace:: :func:`Database.options.set_transaction_causal_read_risky`
+.. |include-port-in-address-database-option| replace:: :func:`Database.options.set_transaction_include_port_in_address`
```

Comment:
It doesn't look like these replacements are used anywhere. The option blurbs could be updated to match other options with database defaults, with the transaction blurb referencing the database function and vice versa, and then these replacements would be used. Also note that if you do this, you'll have to update a couple other files with fake replacements (silly, I know. grep for something like `causal-read-risky-database-option` to find them).

Reply:
Thanks. I think I've addressed this. Still learning about reStructuredText.